package.json

```diff
@@ -17,6 +17,8 @@
     "e2e": "playwright test src/e2e/",
     "e2e:debug": "playwright test src/e2e/ --ui",
     "e2e:smoke": "playwright test src/e2e/ --grep @smoke",
+    "e2e:ui": "playwright test src/e2e/ --grep @ui",
+    "e2e:all": "playwright test src/e2e/",
```

Comment:
I think this is exactly the same as `npm run e2e`? (Alternatively, we could remove that one and rename `e2e:debug` to `e2e:all:debug`, but to me, the "default" one being to run them all feels more intuitive.)

```suggestion
```